web/gds-admin-ui/src/pages/app/dashboard/Tasks.js

```diff
@@ -6,82 +6,112 @@ import { StatusLabel } from 'constants/index';
 import TrisaFavicon from 'assets/images/trisa_favicon.png'
 import CiphertraceFavicon from 'assets/images/ciphertrace.ico'
 import PropTypes from 'prop-types';
+import SimpleBar from 'simplebar-react';
 
 import relativeTime from 'dayjs/plugin/relativeTime'
 import { actionType, useModal } from 'contexts/modal';
 import OvalLoader from 'components/OvalLoader';
 dayjs.extend(relativeTime)
 
 
+const NoData = () => {
+    return (
+        <div className='text-center no-pending'>
+            <h5>No data</h5>
+            <p>
+                No available pending registrations
+            </p>
+        </div>
+    )
+}
 
-
-const Tasks = ({ data, isLoading }) => {
+const PendingReviewsTable = ({ data }) => {
     const [vasp, setVasp] = React.useState({ name: '', id: '' });
-    const history = useHistory()
     const { dispatch } = useModal()
+    const history = useHistory()
 
     const handleResendEmailClick = (name) => {
         setVasp(name)
         dispatch({ type: actionType.SEND_EMAIL_MODAL, payload: { vasp } })
     }
 
     return (
-        <Card style={{ height: '95%' }}>
+        <SimpleBar>
```

Comment:
Could you explain what `SimpleBar` is doing?

Reply:
It's used to add a cross browser scrollbar customization

Reply:
In the case where there are more pending registrations than the height allows for?